text/0092-replay-issue-creation.md

```diff
@@ -17,43 +17,66 @@ These issues can only be detected on the SDK. The Replay back-end will never hav
 
 # Options Considered
 
-### Option 1: SDK Generates Issues Through a Generic Interface
+### Option 1: SDK Creates Issues Through the Error Interface
 
-When the SDK encounters a "replay issue" it will make an HTTP request to a generic interface which will handle the issue creation process.
+When the SDK encounters a "slow click" it will use `sentry.captureException(slowClick)` to create an error issue on the Issues platform.
```

Comment:
It'll be an envelope with an event in it. We could skip the "Issues Platform" bit?

```suggestion
When the SDK encounters a "slow click" it will use `sentry.captureEvent(slowClick)`.
```

Reply:
I think we're saying the same things here.  My extra text is confusing.  Happy to remove it.